composer.json

```diff
@@ -8,14 +8,15 @@
         "php": ">=7.1",
         "barryvdh/laravel-debugbar": "^2.4",
         "friendsofphp/php-cs-fixer": "^2.3",
```

Comment:
I'm getting this warning when I set up locally the project:
```php
Package phpoffice/phpexcel is abandoned, you should avoid using it. Use phpoffice/phpspreadsheet instead.
Package phpunit/phpunit-mock-objects is abandoned, you should avoid using it. No replacement was suggested.
Package guzzle/guzzle is abandoned, you should avoid using it. Use guzzlehttp/guzzle instead.
Package satooshi/php-coveralls is abandoned, you should avoid using it. Use php-coveralls/php-coveralls instead.
```

Please fix it if it's not too much...